pkgs/development/tools/misc/babeltrace/common.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchzip
+, autoreconfHook
+, pkg-config
+, glib
+, elfutils
+
+, version
+, hash
+}:
+
+args: stdenv.mkDerivation (finalAttrs: args // {
+  inherit version;
+
+  src = fetchzip {
+    url = "https://www.efficios.com/files/babeltrace/${finalAttrs.pname}-${finalAttrs.version}.tar.bz2";
```

Comment:
And then you'd be able to use `pname` and `version` without the `finalAttrs.` prefix...

Reply:
I removed `finalAttrs` completely, because it's  not used for anything.